File: autopilot/v1/task_actions/update-quickstart-example-1/update-quickstart-example-1.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$task_actions = $twilio->autopilot->v1->assistants("UAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
                                      ->tasks("tell-a-joke")
                                      ->taskActions()
                                      ->update(array(
                                                   "actions" => array(
                                                       "say" => "I was going to look for my missing watch, but I could never find the time."
                                                   )
                                               )
                                      );

print($task_actions->assistantSid);

File: autopilot/v1/task_actions/update-quickstart-example-1/update-quickstart-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
# DANGER! This is insecure. See http://twil.io/secure
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

task_actions = @client.autopilot
  .assistants('UAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
  .tasks('tell-a-joke')
  .task_actions
  .update(actions: {
       'say' => 'I was going to look for my missing watch, but I could never find the time.'
   })

puts task_actions.assistant_sid
